
string uuid_dataset

int8 num_tiles_x
int8 num_tiles_y
int8 dim_channels
int8[] data